mysite/announcements/views.py

```diff
@@ -3,11 +3,18 @@
 from django.contrib.auth import authenticate, login
 from django import forms
 from .forms import UserRegistrationForm
-from .models import Announcements
+from .models import Announcements, Status
 from django.contrib.auth.decorators import login_required
+from .serializers import StatusSerializer
+from rest_framework import generics
 
 
-@login_required()
+class ListStatusView(generics.RetrieveUpdateAPIView):
+    queryset = Status.objects.all()
+    serializer_class = StatusSerializer
```

Comment:
In what format this accepts data? I don't see any logic where we are updating status of AnnouncementGroup relation.

Reply:
JSON,
RetreiveUpdateView is one of generic views in Django Rest Framework which allows to read or update endpoints

Reply:
What is format of json?

Reply:
JSON is sytax for storing and exchange of data. It's text written with JavaScript object notation